app/components/post_list/thread_overview/thread_overview.tsx

```diff
@@ -25,6 +25,7 @@ type Props = {
     repliesCount: number;
     rootPost?: PostModel;
     testID: string;
+    style: StyleProp<ViewStyle>;
```

Comment:
Should this be optional? (If so, remove it from the tests, which seem weird with the empty object in the snapshot)